### PR TITLE
Routing to survey

### LINKED ISSUE #21: On Click Subscibe if the user has no style the route changes to /style-preferences

### PR DESCRIPTION
**Pull request by relyt4me**
### Type of change made:
- [x] New feature (non-breaking change which adds functionality)
- [ ] Testing - building tests
- [ ] Formatting - no new features
- [ ] Styling- no new features
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Description
* This PR will allow the user to click on a subscribe button from the crates page and go to the new route of style survey

### How should this be manually tested?
* run the app and attempt to click on the subscribe button. Check the route URL has changed and that the page should now say 'Hello World'

### Any background context you want to provide?
* NA

### What are the relevant tickets?
* Resolves #21 

### Checklist
- [x] I have tested this code
- [ ] I have updated the Readme

### CODE REVIEW

File: code/web/src/modules/crate/Item.js [hunk 1]
```diff
@@ -28,35 +29,38 @@ class Item extends PureComponent {
...
+    //   })
+    //   .then(() => {
+    //     this.setState({
+    //       isLoading: false
+    //     })
+
+    //     window.setTimeout(() => {
+    //       this.props.messageHide()
+    //     }, 5000)
+    //   })
```
Comment (jordy1611) [inline code/web/src/modules/crate/Item.js:63]: Keep this for when the user has a style after finishing the survey

**Review by jordy1611 — APPROVED** [review]
Pretty rad code

File: code/web/src/modules/crate/Item.js [hunk 2]
```diff
@@ -28,35 +29,38 @@ class Item extends PureComponent {
     }
   }
 
+  // Original add of crate commented out
+  // Currently just navigates to the survey page
   onClickSubscribe = (crateId) => {
```
Comment (Leasw144) [inline code/web/src/modules/crate/Item.js:34]: To sum up, we're repurposing this method to take us to the survey page, right?

**Comment by Leasw144** [comment]
Thanks for driving this round, great job!

**Pull request merged**